src/standalone/Soil/soil_hydrology_parameterizations.jl

```diff
@@ -392,3 +396,101 @@ function viscosity_factor(T::FT, γ::FT, γT_ref::FT) where {FT}
     Theta = FT(exp(factor))
     return Theta
 end
+
+"""
+    ice_fraction(θ_l::FT, θ_i::FT, ν::FT, θ_r::FT, ρ_l::FT, ρ_i::FT)::FT 
+where {FT}
+
+Computes and returns the
+fraction of the humidity in the pore space 
+that is due to sublimation.
+
+f = θ_iρ_i/(θ_iρ_i+(θ_l-θ_r)ρ_l)
+"""
+function ice_fraction(
+    θ_l::FT,
+    θ_i::FT,
+    ν::FT,
+    θ_r::FT,
+    ρ_l::FT,
+    ρ_i::FT,
+)::FT where {FT}
+    m_l = (θ_l - θ_r) * ρ_l
+    m_i = θ_i * ρ_i
+    f = m_i / (m_i + m_l)
+    return f
+end
+
+"""
+    soil_tortuosity(θ_l::FT, θ_i::FT, ν::FT) where {FT}
+
+Computes the tortuosity of water vapor in a porous medium,
+as a function of porosity `ν` and the volumetric liquid water
+and ice contents, `θ_l` and `θ_i`.
+
+See Equation (1) of : Shokri, N., P. Lehmann, and
+D. Or (2008), Effects of hydrophobic layers on evaporation from
+porous media, Geophys. Res. Lett., 35, L19407, doi:10.1029/
+2008GL035230.
+"""
+function soil_tortuosity(θ_l::FT, θ_i::FT, ν::FT) where {FT}
+    safe_θ_a = max(ν - θ_l - θ_i, eps(FT))
+    return safe_θ_a^(FT(2.5)) / ν
+end
+
+"""
+    soil_resistance(θ_l::FT,
+                    θ_i::FT,
+                    ice_frac::FT,
+                    hydrology_cm::C,
+                    ν::FT,
+                    θ_r::FT,
+                    d_ds::FT,
+                    earth_param_set::EP,
+                   ) where {FT, EP, C}
+
+Computes the resistance of the top of the soil column to
+water vapor diffusion, as a function of the surface
+volumetric liquid water fraction `θ_l`, the augmented
+liquid water fraction `ϑ_l`,  the volumetric ice water
```

Comment:
It would be helpful to make sure all the function parameters are clearly identified in the docstring. For example, I don't understand how `ϑ_l` enters here, or what `d_ds` is.